Formula/tsduck.rb

```diff
@@ -0,0 +1,25 @@
+class Tsduck < Formula
+  desc "MPEG Transport Stream Toolkit"
+  homepage "https://tsduck.io/"
+  url "https://github.com/tsduck/tsduck/archive/v3.28-2551.tar.gz"
+  sha256 "ff39c9b97e17a5ae06556a119e24783427a86b1bb5ac75c7ba0340c7bd07d02d"
+  license "BSD-2-Clause"
+  head "https://github.com/tsduck/tsduck.git", branch: "master"
+
+  depends_on "dos2unix" => :build
+  depends_on "gnu-sed" => :build
+  depends_on "grep" => :build
+  depends_on "openjdk" => :build
```

Comment:
Are you sure this is build only?

Reply:
Yes. The JDK is used to build a .jar for the Java binding. The Java bindings are used by users who develop Java applications on top of TSDuck. But it cannot be a mandatory requirement for installation. Most users won't develop Java application.

I do not know why the Java label was added. This is a C++ project with Java and Python bindings for concerned users.